src/iOS/toga_iOS/widgets/imageview.py

```diff
@@ -19,6 +20,9 @@ def startup(self):
         self._impl.setTranslatesAutoresizingMaskIntoConstraints_(False)
         self._impl.setAutoresizesSubviews_(False)
 
+        # Add the layout constraints
```

Comment:
Hope it is fine now. 

There are some commented properties (scaling, alignment) that I am not sure if they are old or are pending to implement. I din't remove them just in case.